src/common/motor.rs

```diff
@@ -129,16 +134,24 @@ where
     fn set_power(&mut self, pct: f64) -> anyhow::Result<()> {
         self.lock().unwrap().set_power(pct)
     }
+    fn stop(&mut self) -> anyhow::Result<()> {
+        self.lock().unwrap().stop()
+    }
 }
 
 impl Motor for FakeMotor {
+    fn get_position(&mut self) -> anyhow::Result<i32> {
+        Ok(self.pos as i32)
+    }
     fn set_power(&mut self, pct: f64) -> anyhow::Result<()> {
         info!("setting power to {}", pct);
         self.power = pct;
         Ok(())
     }
-    fn get_position(&mut self) -> anyhow::Result<i32> {
-        Ok(self.pos as i32)
+    fn stop(&mut self) -> anyhow::Result<()> {
+        info!("stopping motor");
```

Comment:
make it debug! or remove it (and if you could do the same for the statement in set_power :P)